Otsu_Thresholding.py

```diff
@@ -0,0 +1,65 @@
+import matplotlib.pyplot as plt
+from skimage.io import imread
```

Comment:
If it is possible refrain from using the `from module import function` syntax. While it works like a charm, it is harder to comprehend.
`thresh = threshold_otsu(image)`
Take this for example, without knowing that you imported this function explicitly from `skimage.filters` I would expect that you wrote this function for yourself.